paper/README.qmd

```diff
@@ -1,160 +1,202 @@
+## Mathematical Framework
 
-> we suggest that a similar approach based on the percentage of the reference source length rather than the tested source might be more related to generalization...
+### Core Variables and Parameters
 
-- Our algorithm, due to its reporting of shared overlap permits us to adapt the Goodchild algorithm and use % source length overlap as a cut off. 
-  - The issue here is that there may be complete overlap from the source but the source may continue well past. we must address this. This is why we will lean on areal weighted interpolation 
-  
-- Java Conflation Suite (JCS, 2003)
+The ANIME algorithm operates on two sets of linestring features with the formal definitions summarized in @tbl-variables-parameters.
 
-- delimited strokes algorithm 2009 zhang
+| Variable/Parameter           | Symbol    | Description                                                                    |
+| ---------------------------- | --------- | ------------------------------------------------------------------------------ |
+| Source features              | $A$       | The input `FeatureCollection` from which attributes are transferred           |
+| Target features              | $B$       | The input `FeatureCollection` to which features are being matched             |
+| Distance Tolerance           | $DT$      | The maximum Euclidean distance for segment matching (in map units)            |
+| Angle Tolerance              | $AT$      | The maximum angle difference (degrees) for parallel segment consideration     |
+| Shared Length                | $SL_{ij}$ | The length of geometric overlap between source feature $i$ and target feature $j$ |
+| Source-weighted proportion   | $sw_{ij}$ | The proportion of source feature $i$ matched to target feature $j$: $SL_{ij} / \text{length}(i)$ |
+| Target-weighted proportion   | $tw_{ij}$ | The proportion of target feature $j$ matched to source feature $i$: $SL_{ij} / \text{length}(j)$ |
 
-- Kim et al., 2017 creates an algorithm to conflate historic road network with new ones
-  - uses a number of spatial similarity measures with c4.5 decision tree to classify 
-  - "linear directional mean" of each line segment in a LineString
-    - challenging for longer linestrings with many segments
-  - shorter line median Hausdorff distance (SMHD)
-  - absolute value of cosine similarity (aCS)
-    - note we do something very similar
-  
-  
-- (Chehreghan & Abbaspour, 2018) use a genetic algorithm. These are computational expensive and do not sacle well
+: Summary of key variables and parameters {#tbl-variables-parameters}
 
-- Esri conflation toolset: buffer analysis and similarity     
 
-- overline approach (morgan and lovelace)
+Each LineString in datasets $A$ and $B$ is formally decomposed into its component line segments:
 
+- $A_{ik}$: The $k$-th line segment of the $i$-th LineString in source dataset $A$
+- $B_{jl}$: The $l$-th line segment of the $j$-th LineString in target dataset $B$
```

Comment:
I'm not comfortable with the changing of the notation here. I think the subscript should remain `ik` and `jk` OR be even more specific with `Source_{ik}` and `Target_{ik}`. Having different letters represent the same thing makes it more difficult to comprehend.